operator/internal/custom/istio_client.go

```diff
@@ -15,8 +16,10 @@ import (
 )
 
 const (
-	firstElementIdx = 0
-	contractVersion = "1"
+	firstElementIdx    = 0
+	contractVersion    = "1"
```

Comment:
use `v1` is more fit for version definition.

Reply:
correct, please change the template accordingly

Reply:
Applied it [here](https://github.com/kyma-project/lifecycle-manager/pull/231/commits/10183a46319e726d4351465c872a45443ebd123e)